stats/B161/feedback-analysis.ipynb

```diff
@@ -0,0 +1,1130 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# MI-PYT feedback"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Feedback for the [MI-PYT course](https://github.com/cvut/MI-PYT) was requested through a [feedback form application](https://github.com/encukou/feedback-form). The results are [public](results.csv).\n",
```

Comment:
not sure if the relative link works on GitHub render, it appears to not work when I look at your repo